### PR TITLE
Stats adjustments

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

File: src/main/resources/templates/StatusResource/issues.html [hunk 1]
```diff
@@ -15,114 +15,114 @@
...
 		<div class="ui icon message">
 			<i class="small sync icon"></i>
 			<div class="content">
-				<p>Updated on {stats.updated.formatDateTime} (refreshed every 30 minutes).</p>
+				<p>Updated {stats.updated.formatDateTime} (refreshed every 6 hours).</p>
 			</div>
 		</div>
 
 		<div class="ui message">
-        	<canvas id="canvas-stacked"></canvas>
```
Comment (gsmet) [inline src/main/resources/templates/StatusResource/issues.html:31]: I reformatted things with tabs to be consistent with the other HTML files.

**Comment by gsmet** [comment]
@rsvoboda once I have your blessing, I will push this to production.

**Review by rsvoboda — APPROVED** [review]
Approving, works fine on machine, no issues spotted in code.

@gsmet so you dropped `initialize(@Observes StartupEvent startupEvent)` because `@Scheduled(every = "10m")  public void updateStatus()` runs at boot too and we had doubled initialization of the data.

And DCL was introduced to ensure correct data are returned if user hit the endpoint righ after the boot but before `updateStatus()` finishes.

Is it correct conclusion?

**Comment by gsmet** [comment]
I **think** the scheduled methods run at startup somehow. But even if not, I wanted to be extra sure thus the DCL.

The thing is that with this new setup, we have a fast startup and we are sure the user will have data.

**Pull request merged**